2018-08-31 13:34:57 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 13:34:57 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 13:34:57 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 13:34:57 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 13:34:57 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 13:34:57 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 13:34:57 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 13:34:57 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 13:34:57 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 13:34:57 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 14:26:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 14:26:08 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 14:26:08 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 14:26:08 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 14:26:08 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 14:26:08 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 14:26:08 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 14:26:08 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 14:26:08 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 14:26:08 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 14:35:07 jdbc[3]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 14:35:07 jdbc[4]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 14:35:07 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 14:35:07 jdbc[6]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 14:35:07 jdbc[7]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 14:35:07 jdbc[8]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 14:35:07 jdbc[9]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 14:35:07 jdbc[10]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 14:35:07 jdbc[11]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
2018-08-31 14:35:07 jdbc[12]: exception
org.h2.jdbc.JdbcSQLException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-196]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1484)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:471)
	at org.h2.jdbc.JdbcConnection.checkClosedForWrite(JdbcConnection.java:1469)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:498)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:451)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:78)
	at org.apache.aries.tx.control.jdbc.xa.connection.impl.XAConnectionWrapper.close(XAConnectionWrapper.java:54)
	at com.zaxxer.hikari.pool.PoolBase.quietlyCloseConnection(PoolBase.java:116)
	at com.zaxxer.hikari.pool.HikariPool$1.run(HikariPool.java:415)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
